pymc/tests/distributions/test_distribution.py

```diff
@@ -0,0 +1,499 @@
+#   Copyright 2020 The PyMC Developers
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+import functools
+import warnings
+
+import aesara
+import aesara.tensor as at
+import numpy as np
+import numpy.random as npr
+import pytest
+import scipy.stats
+import scipy.stats.distributions as sp
+
+from aeppl.logprob import ParameterValueError
+from aesara.graph.basic import ancestors
+from aesara.tensor.random.op import RandomVariable
+from aesara.tensor.var import TensorVariable
+from numpy.testing import assert_almost_equal
+from scipy import integrate
+
+import pymc as pm
+
+from pymc.distributions import (
+    CAR,
+    Binomial,
+    Constant,
+    ExGaussian,
+    MvNormal,
+    MvStudentT,
+    Normal,
+    TruncatedNormal,
+    joint_logp,
+    logp,
+)
+from pymc.distributions.continuous import get_tau_sigma
+from pymc.distributions.shape_utils import to_tuple
+from pymc.model import Model
+from pymc.tests.distributions.util import Nat, R, Rplus, Rplusbig, Vector, check_logp
+from pymc.tests.helpers import select_by_precision
+from pymc.vartypes import continuous_types
+
+
+def integrate_nd(f, domain, shape, dtype):
```

Comment:
I think we are not using this helper anymore, so we can remove it (separate commit) or move it to `util`